Makefile

```diff
@@ -85,3 +85,11 @@ docker-internal-linkcheck:
 container-internal-linkcheck: link-checker-image-pull
 	$(CONTAINER_RUN) $(CONTAINER_IMAGE) hugo --config config.toml,linkcheck-config.toml --buildFuture
 	$(CONTAINER_ENGINE) run --mount type=bind,source=$(CURDIR),target=/test --rm wjdp/htmltest htmltest
+
+clean-api-reference:
+	rm -rf content/en/docs/reference/kubernetes-api
+
+api-reference: clean-api-reference ## Build the API reference pages. go needed
+	mkdir content/en/docs/reference/kubernetes-api
+	cd api-ref-generator && \
+		go run cmd/main.go kwebsite --config-dir config/v1.19/ --file api/v1.19/swagger.json --output-dir ../content/en/docs/reference/kubernetes-api --templates templates
```

Comment:
Can we look up the release version based on (eg) `config.toml`? I'd prefer to not hard code this - even if that change delays a merge.

Reply:
It would be possible, but is it important to make this change now? This built target needs to change but I don't see this as a requirement to merge.